### ISSUE TITLE
Running "python -m swaggerconformance <RESOURCE>" doesn't work

### ISSUE THREAD

**Issue by alexwlchan** [olipratt/swagger-conformance#2]
If I run the example provided in the README against a Swagger URL, I get an error:

```console
$ python -m swaggerconformance 'https://api.wellcomecollection.org/catalogue/v1/swagger.json'
usage: python -m swaggerconformance [-h] [-n N] schema_path
python -m swaggerconformance: error: unrecognized arguments: https://api.wellcomecollection.org/catalogue/v1/swagger.json
```

The issue is in `__main__.py`: you're passing `sys.argv` into `main()`, which includes everything – including the name of the running program. Better would be to use `sys.argv[1:]`.

Note that if you don’t pass anything into `ArgumentParser.parse_args`, I think it picks the correct arguments for you.

I’d submit a patch, but I’m having trouble constructing a URL that I can pass as `schema_path` to verify it’s working correctly in the tests.

---

```console
$ pip freeze
attrs==17.4.0
certifi==2018.1.18
chardet==3.0.4
coverage==4.5.1
hypothesis==3.55.4
idna==2.6
pyaml==17.12.1
pyswagger==0.8.37
PyYAML==3.12
requests==2.18.4
six==1.11.0
swagger-conformance==0.2.2
urllib3==1.22
validate-email==1.3

$ python --version
Python 3.6.1
```

**Comment by olipratt** [comment]
Hi Alex!

Thanks for taking the time to raise this 😄 

This is strange, because the whole reason this project even exists was that after I learnt about Hypothesis from the talk you gave at an Engineering Conference a few years ago, I wanted to find a way to give it a try, and this was my idea for a way to try it out. So thanks for that too!

The reason I passed the arguments in like that was specifically so I could call that function from the test code with arguments to test it. Of course, then one of the handful of lines that I mark as not requiring code coverage testing is the one with the bug in it... sigh, that will teach me.

I'll make a fix, it's trivial and as you suggest.

I think from your other issue you were having trouble setting up a test server. To setup a quick test, the `examples/` directory has an example server in it, so if you checkout the project and then in two terminals run e.g.:

-  `cd examples && python3 datastorapi.py`
- `python3 -m swaggerconformance http://127.0.0.1:5000/api/schema`

This should work and exercise the schema for that simple server (though in doing that myself I spotted another quick bug which I'll fix in the same commit).
